src/frontend/tsx/GamePanel.tsx

```diff
@@ -226,18 +227,17 @@ export function GamePanel () {
         {hasGame
           ? <button
             className='option'
-            disabled={!hasGame}
             onClick={() => setDisplayGame(null)}>
             Practice Board</button>
           : <button
             className='option'
-            disabled={hasGame || !practiceHasMoved}
+            disabled={!practiceHasMoved}
             onClick={() => setPracticeBoard(null)}>
             Reset Practice Board</button>
         }
```

Comment:
The "reset practice board" button can never actually appear here, that's in the `PracticePanel` now. This should be:
```
{/* practice board */}
<button
  className='option'
  onClick={() => setDisplayGame(null)}>
  Practice Board</button>
```